### PR TITLE
Support skiplagged as web source

### PR DESCRIPTION
**Pull request by samuraitruong**
This is my initial commit, I have cleanup code & naming convention , you can merge it now, or give me feedback then I will finalize it later tomorrow after work

### CODE REVIEW

File: PogoLocationFeeder/Repository/SkiplaggedPokemonRepository.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    list.Add(sniperInfo);
+                }
+            }
+            return list;
+        }
+
+        private SkiplaggedSniperInfo Map(pokemon result)
+        {
+            var sniperInfo = new SkiplaggedSniperInfo();
+            var pokemonId = PokemonParser.ParsePokemon(result.pokemon_name);
```
Comment (ExcuseMi) [inline PogoLocationFeeder/Repository/SkiplaggedPokemonRepository.cs]: If you have the pokemonId, please use PokemonParser.ParseById instead the name.

**Comment by ExcuseMi** [comment]
Hi, thanks for  the PR.
Isn't calling skiplagged for each location in parallel a little bit aggressive?
I would rather have a background thread that scans one of the locations in the list every 1 to 2 seconds and sends back the result when the FindAll is called on the RarePokemonRepository.

**Comment by samuraitruong** [comment]
Hi,

I don't mind about run in parallel because the request to skiplagged is very fast.

That works perfectly on my local. I will review and update  your feedback. thanks
Truong

________________________________

**Comment by ExcuseMi** [comment]
I'm afraid for the skiplagged server

**Comment by samuraitruong** [comment]
yeah, I will change it to main thread and fetch every 5 sec for each location.

________________________________

**Pull request closed without merging**